docs/api/app.md

```diff
@@ -331,6 +331,10 @@ to the npm modules spec. You should usually also specify a `productName`
 field, which is your application's full capitalized name, and which will be
 preferred over `name` by Electron.
 
+### `app.setName()`
```

Comment:
I think the `name` parameter to this function should be documented and mention that it should be a `String`

The docs for `app.setPath` have an example of the format, https://github.com/k0kubun/electron/blob/app-doc-set-name/docs/api/app.md#appsetpathname-path